code/__DEFINES/obj_flags.dm

```diff
@@ -2,43 +2,43 @@
 
 
 #define EMAGGED					(1<<0)
-#define IN_USE					(1<<1) // If we have a user using us, this will be set on. We will check if the user has stopped using us, and thus stop updating and LAGGING EVERYTHING!
```

Comment:
this is a whole lot of pointless whitespace changes

Reply:
it bothered me that they werent all in line ok